sdk/python/kfp/v2/compiler/compiler.py

```diff
@@ -172,11 +172,36 @@ def _create_pipeline(
 
     return pipeline_spec
 
+  def _create_pipeline_job(
+      self,
+      pipeline_spec: pipeline_spec_pb2.PipelineSpec,
+      pipeline_root: Optional[str] = None,
```

Comment:
IIUC pipeline_root is still required for now?

Reply:
Updated.